kuma/users/signup.py

```diff
@@ -24,31 +24,20 @@ class SignupForm(BaseSignupForm):
     This overrides the default error messages for the username form field
     with our own strings.
 
-    It has an additional other_email form field to handle the case of GitHub
-    which may deliver a number of emails for users to choose from upon signup.
-
     The heavy lifting happens in the view.
     """
 
-    email = forms.CharField(
-        required=False, widget=forms.TextInput(attrs={"type": "email"})
-    )
```

Comment:
Removed because now it *is* required and what's left if you change that is that this is just want the parent class set already.